perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -1251,10 +1251,10 @@ sub waste_payment_type {
     my ($self, $type, $ref) = @_;
 
     my ($sub_type, $category);
-    if ( $type eq 'Payment: 01' ) {
+    if ( $type eq 'Payment: 01' || $type eq 'First Time' ) {
         $category = 'Garden Subscription';
         $sub_type = $self->waste_subscription_types->{New};
-    } elsif ( $type eq 'Payment: 17' ) {
+    } elsif ( $type eq 'Payment: 17' || $type eq 'Regular' ) {
```

Comment:
Should the tests continue to include `Payment: 17` and `Payment: 01` too?

Reply:
I'm not sure there ever used in practice but given the docs and sample data disagree it seemed best to hedge, so yes.